recipes/etl/all/conanfile.py

```diff
@@ -60,7 +60,7 @@ def _module_file_rel_path(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "etl")
-        self.cpp_info.set_property("cmake_target_name", "etl")
+        self.cpp_info.set_property("cmake_target_name", "etl::etl")
```

Comment:
```suggestion
        self.cpp_info.set_property("cmake_target_name", "etl::etl")
        self.cpp_info.set_property("cmake_target_aliases", ["etl"]) # legacy target before 20.27.2
```

It might be a good idea to still provide legacy imported target (which was `etl` before 20.27.2, see https://github.com/ETLCPP/etl/pull/523) to not break users.

Reply:
Thanks, I was thinking the same with my first attempt: calling set_property twice, which wasn't correct.